src/minimasonry.js

```diff
@@ -91,6 +92,13 @@ MiniMasonry.prototype.getCount = function() {
         return Math.floor((this._width - this._currentGutterX) / (this.conf.baseWidth + this._currentGutterX));
     }
 
+    if (this.conf.numberOfColumnsOnMobile && this.conf.numberOfColumnsOnMobile > 1 && this._width < 620) {
```

Comment:
You and I did some work together for a while and I'm now noticing you didn't implement the code we worked on together. Instead of using the hard-coded value of 620 we used the calculated value, remember?...

Reply:
Oh... I thought that's why we had to fork it to fit our need, as the calculation was being rigid.

I tried making it work, but it wasn't giving the desired result. I will spend more time on it now to make sure it works.

Reply:
Are you sure here? `this._width < (this.conf.baseWidth + (2 * this._currentGutterX))` seems like it is doing exactly what we want and we should figure out why.

Try logging the value of `this.conf.baseWidth + (2 * this._currentGutterX)` to see what it is compared to your number.